packages/ui/Slottable/Slottable.js

```diff
@@ -62,6 +62,8 @@ const Slottable = hoc(defaultConfig, (config, Wrapped) => {
 			const prop = props[slot];
 			if (prop && dist) {
 				distributed[slot] = [].concat(prop, dist);
+			} else if (prop) {
+				distributed[slot] = prop;
```

Comment:
Why is this necessary?  Don't we spread `props` below to achieve this same effect?

Reply:
that's fair!